tests/suite/zjson/empty-records.yaml

```
@@ -0,0 +1,11 @@
+zql: '*'
+
+input: |
+  {"id":23,"type":[],"values":[]}
+  {"id":24,"type":[{"name":"ja3s","type":[]}],"values":[[]]}
+
+output-format: zjson
+output: |
+    {"id":23,"type":[],"values":[]}
+    {"id":24,"type":[{"name":"ja3s","type":[]}],"values":[[]]}
+
```

Nits: Ditch the extra blank line. Indent this block scalar by two spaces as we do elsewhere.
```suggestion
  {"id":23,"type":[],"values":[]}
  {"id":24,"type":[{"name":"ja3s","type":[]}],"values":[[]]}
```